metaseq/generation_metrics/__init__.py

```diff
@@ -0,0 +1,19 @@
+import importlib
+import os
+
+from .coco_metrics import CocoMetrics
+from .hf_metrics import HFEvaluateMetrics
+from .metrics import GenerationMetrics
+from .parlai_metrics import ParlAiMetrics
+
+__all__ = ["GenerationMetrics", "HFEvaluateMetrics", "ParlAiMetrics", "CocoMetrics"]
```

Comment:
I think we can remove this and everything below. We're not including the `tasks` directory in this PR, and it's also no longer used.